style.css

```diff
@@ -184,21 +190,90 @@ body {
   flex-grow: 0;
 }
 
+#fb-title {
+  margin-left: 2rem;
+}
+
+#fb-info {
+  margin-left: 2rem;
+  margin-top: 1rem;
+}
+
 .frame2-img {
   width: 236px;
   height: 32px;
   padding: 0.5rem;
-  margin-top: -30px;
+  margin-top: -0.9px;
   margin-bottom: 0;
   margin-right: 0;
-  margin-left: -50px;
+  margin-left: -0.2px;
+}
+
+.frame3-img {
+  display: flex;
+  align-items: center;
+}
+
+.project-info {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  font-size: 16px;
+  margin-top: 1.5rem;
+  margin-bottom: 1rem;
+  font-weight: bold;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+  order: 2;
+}
```

Comment:
- [OPTIONAL] This is the final design of your portfolio project so your project is meant to look good on desktop and mobile. It seems like the project cards are not properly aligned in the center in the mobile version, please look into this as your design is meant to be a replica of the Figma design.Thanks :smile: 

- Screenshot for reference.
![Screenshot from 2023-04-08 11-20-40](https://user-images.githubusercontent.com/107632780/230716231-79f4b94f-7c73-46a3-bb56-30fadaf0dba3.png)